api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -7,7 +7,8 @@
 from django.test import TransactionTestCase
 
 from scpca_portal import common, metadata_file
-from scpca_portal.management.commands.load_data import Command
+from scpca_portal.management.commands.configure_aws_cli import Command as configure_aws_cli
```

Comment:
We moved the calling of the `configure_aws_cli` command outside of `run_tests.sh` and inside of `test_load_data` due to a permissions error accessing the `~/.aws/config` file (first noted on this GHA https://github.com/AlexsLemonade/scpca-portal/actions/runs/10182489658/job/28165155525?pr=809)

We should open a ticket about this to investigate what happened here.

Reply:
Yeah let's open the ticket and remove it from being executed before the tests for now.